### PR TITLE
tail: add page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

`tail.md` in macOS doesn't support long arguments so this page is created.

### CODE REVIEW

**Comment by blueskyson** [comment]
There is no need to create this page because there already exists `common/tail.md` with almost the same usages.

**Review by blueskyson — APPROVED** [review]

File: pages/osx/tail.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+> Display the last part of a file.
+> See also: `head`.
+> More information: <https://manned.org/man/freebsd-13.0/tail.1>.
+
+- Show last 'count' lines in file:
+
+`tail -n {{count}} {{path/to/file}}`
+
+- Show all file since line 'count':
```
Comment (navarroaxel) [inline pages/osx/tail.md:11]:
```suggestion
- Print a file from a specific line number:
```

File: pages/osx/tail.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`tail -n +{{count}} {{path/to/file}}`
+
+- Show last 'count' bytes in file:
```
Comment (navarroaxel) [inline pages/osx/tail.md:15]:
```suggestion
- Print a specific count of bytes from the end of a given file:
```

File: pages/osx/tail.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`tail -c {{count}} {{path/to/file}}`
+
+- Keep reading file until `Ctrl + C`:
```
Comment (navarroaxel) [inline pages/osx/tail.md:19]:
```suggestion
- Print the last lines of a given file and keep reading file until `Ctrl + C`:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**